portfolio/src/main/webapp/script.js

```diff
@@ -66,8 +66,13 @@ function makeLarger(pic){
  * fetches comments from data servlet
  */
 function getComments(){
-    fetch('/data').then(response => response.json()).then((comments) => {
     const commentsContainer = document.getElementById('comments-container');
+    commentsContainer.innerHTML = "";
+
+    var commentLimit = getCommentLimit();
+    var fetchURL = '/data?comment-limit='+commentLimit;
```

Comment:
Supernit: spaces around + symbol

```suggestion
    var fetchURL = '/data?comment-limit=' + commentLimit;
```